plugins/grafana-custom-plugins/grafana-sankey-plugin/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "antreaflowvisibility-grafana-sankey-plugin",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "grafana-toolkit plugin:build",
+    "test": "grafana-toolkit plugin:test",
+    "dev": "grafana-toolkit plugin:dev",
+    "watch": "grafana-toolkit plugin:dev --watch",
+    "sign": "grafana-toolkit plugin:sign",
+    "start": "yarn watch"
+  },
+  "author": "antreaflowvisibility",
```

Comment:
I don't mind keeping the author as it is, but we can consider using either 'theia' or 'antrea-io' (the github org name).
What's your opinion?

Reply:
changed to 'antrea-io'